src/util.ts

```diff
@@ -38,7 +38,7 @@ export function docstring(parts: ts.SymbolDisplayPart[]): string {
  * Normalizes path to match POSIX standard (slashes)
  */
 export function normalizePath(file: string): string {
-	return file.replace(new RegExp('\\' + path.sep, 'g'), path.posix.sep);
+	return file.replace(new RegExp('[\/\\\\]', 'g'), path.posix.sep);
```

Comment:
Unix allows backslashes as valid characters in file names, so this is incorrect
```
$ touch \\.txt
$ ls -la
-rw-r--r--     1 felix  staff       0 Mar 15 10:32 \.txt
```

Also equivalent to `return file.replace(/\\/g, '/')` (no need to replace forward slashes with forward slashes, no need to use the regexp constructor and `path.posix.sep` is always the forward slash)